ts/components/messages/MessageDetailData.tsx

```diff
@@ -96,7 +94,9 @@ class MessageDetailData extends React.PureComponent<Props> {
       <View style={styles.container}>
         <Text>
           {I18n.t("messageDetails.dateSending")}
-          <Text bold={true}>{` ${this.date} - ${this.time}`}</Text>
+          <Text bold={true}>{` ${convertDateTimeToWordDistance(
```

Comment:
Since we are introducing the messages paginated, I guess this change should be addressed also here in the new component
https://github.com/pagopa/io-app/blob/5fc683df0c836919d287210c7778637eaa34bdda/ts/components/messages/paginated/MessageDetail/Content.tsx#L59

Reply:
Thanks for the advice, changed through 083e705